node/blockchain/serializers/block.py

```diff
@@ -11,6 +11,14 @@
 
 
 class BlockSerializer(serializers.ModelSerializer):
+    """
+    Block are structure that store data. These data describe changes to the network and originate
+    from signed change requests, such as:<br>
+    * Genesis;<br>
```

Comment:
let's generate this list from code, otherwise we will be forgetting to update it